packages/mobile/src/app/App.tsx

```diff
@@ -54,6 +54,34 @@ export class App extends React.Component {
         appStartListener.remove()
       }
     )
+
+    Linking.addEventListener('url', this._handleOpenURL)
+    const url = await Linking.getInitialURL()
+
+    if (url) {
+      await sleep(2000)
```

Comment:
Why the sleep here?

Reply:
That was me attempting to do https://github.com/celo-org/celo-monorepo/issues/351.

After startup, the app determines what the first screen should be. I was trying to wait for the app to do that here, and then navigate to the auth screen. Seems to be hit or miss